v2/driver/driver.go

```diff
@@ -183,24 +192,186 @@ func Container(ctx context.Context, opts Options, mlmd *metadata.Client) (execut
 		Inputs:  inputs,
 		Outputs: provisionOutputs(dag.GetPipelineRoot(), opts.Task.GetTaskInfo().GetName(), opts.Component.GetOutputDefinitions()),
```

Comment:
@Bobgy I'm wondering why we construct output artifact here in driver and construct output parameter in launcher https://github.com/kubeflow/pipelines/blob/ed3453f7940b7f95e6c0ef02f44626f8eb55cdbf/v2/component/launcher_v2.go#L81? Can we move all these logic to launcher?

Reply:
done